conf/script/ha_setup

```diff
@@ -60,6 +60,9 @@ config() {
     # Decision Maker conf
     cp -rf ${SOURCE_HA_CONF}/decision_monitor_conf_vm.json ${HA_CONF}/decision_monitor_conf.json
     # config_decision_maker
+
+    # Update node info in the ha.conf file
+    python3 ${BASE_DIR}/node_info_update.py
```

Comment:
why the node information is being stored in conf file ?

Reply:
@ajaysrivas  Sorry, for the delayed response as I was working on other bug tickets related to sspl from the past two weeks.

I took reference of EOS-12738 and below LLD doc and raised PR to confirm the approach of the implementation. If you think it's not required then we can figure out another way to store node related information.

https://seagatetechnology-my.sharepoint.com/:w:/r/personal/ajay_srivastava_seagate_com/_layouts/15/doc2.aspx?sourcedoc=%7BCB9F742E-E2F3-4A62-8DAB-3C02A0539100%7D&file=HA%20LLD%20Ideas%20-%20VM%20%26%20JBOD.docx&action=default&mobileredirect=true&cid=43df06e5-e680-46a9-8c71-9b28184c82e4

@indrajitzagade @ajay-paratmandali  Your thoughts, please.